README.md

```diff
@@ -9,7 +9,7 @@ Provides an object oriented PHP wrapper for the [Twitter API](https://developer.
 Run the following to install this library:
 
 ```bash
-$ composer require laminas/laminas-twitter
```

Comment:
Our rules allow this. We use the leading `$` to indicate a command to execute, and subsequent lines show either the output or continuations.